pkg/scheduler/framework/plugins/registry.go

```diff
@@ -65,8 +59,8 @@ func NewInTreeRegistry(args *RegistryArgs) framework.Registry {
 		noderesources.LeastAllocatedName:           noderesources.NewLeastAllocated,
 		noderesources.RequestedToCapacityRatioName: noderesources.NewRequestedToCapacityRatio,
 		noderesources.ResourceLimitsName:           noderesources.NewResourceLimits,
-		volumebinding.Name: func(_ *runtime.Unknown, _ framework.FrameworkHandle) (framework.Plugin, error) {
-			return volumebinding.NewFromVolumeBinder(args.VolumeBinder), nil
+		volumebinding.Name: func(_ *runtime.Unknown, fh framework.FrameworkHandle) (framework.Plugin, error) {
```

Comment:
we don't need to have this wrapper function, here we should just do 
```volumebinding.Name: volumebinding.New``` like the other plugins.

In VolumeBinding plugin, change [```NewFromVolumeBinder```](https://github.com/kubernetes/kubernetes/blob/9822016bf8f46d434087e05d30af5f5ffa8bc120/pkg/scheduler/framework/plugins/volumebinding/volume_binding.go#L101) to 

```
func New(_ *runtime.Unknown, fh framework.FrameworkHandle) (framework.Plugin, error) {
    return &VolumeBinding{
	binder: fh.VolumeBinder(),
    }, nil
}```

Reply:
Sry that misunderstanding the requirement, have updated. Thanks for the review!